.gitignore

```diff
@@ -7,6 +7,5 @@ coverage
 api-docs
 **/*.tgz
 **/dist*
-**/package
```

Comment:
Why are we removing this?

Reply:
We don't have any files named just `package` do we?

Reply:
I think it's generated when we do a release.